src/App.elm

```diff
@@ -51,15 +50,9 @@ type alias Model =
     }
 
 
-init : Flags -> Url -> Nav.Key -> ( Model, Cmd Msg )
-init flags url navKey =
+init : Env -> Route -> Nav.Key -> ( Model, Cmd Msg )
+init env route navKey =
```

Comment:
App is being wrapped by the new Main module that ensures there's a codebase hash and a perspective before starting up the app.